### ISSUE TITLE
batch_min_rows argument in azurerm_stream_analytics_output_blob can has a range of 1-10000 while the ui allows for 200000

### ISSUE THREAD

**Issue by Juliavol** [hashicorp/terraform-provider-azurerm#22292]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

1.4.0

### AzureRM Provider Version

3.5.9

### Affected Resource(s)/Data Source(s)

azurerm_stream_analytics_output_blob

### Terraform Configuration Files

```hcl
resource "azurerm_stream_analytics_output_blob" "output" {
  name                      = "${var.env}-output-to-blob-storage"
  stream_analytics_job_name = "cyber-data-export-job-${var.env}"
  resource_group_name       = data.azurerm_resource_group.rg.name
  storage_account_name      = azurerm_storage_account.data_lake_storage.name
  storage_account_key       = azurerm_storage_account.data_lake_storage.primary_access_key
  storage_container_name    = azurerm_storage_container.blob_container.name
  path_pattern              = "cyber-data/{partition_key}/{date}/{time}"
  date_format = "yyyy-MM-dd"
  time_format = "HH"
  batch_min_rows  = 200000
  batch_max_wait_time = "01:00:00"
  
  
  serialization {
    type            = "Parquet"
  }
}
```

### Debug Output/Panic Output

```shell
│ Error: expected batch_min_rows to be in the range (0 - 10000), got 200000
│ 
│   with module.test-adb.azurerm_stream_analytics_output_blob.output,
│   on ../../modules/databricks/streamAnalyticsJob.tf line 52, in resource "azurerm_stream_analytics_output_blob" "output":
│   52:   batch_min_rows  = 200000
│
```

### Expected Behaviour

200000is a value that should be allowed as t is allowed in the UI

### Actual Behaviour

the resource does not get created due to this limit

### Steps to Reproduce

terraform apply

**Comment by jiaweitao001** [comment]
Hi @Juliavol , thanks for opening the issue. I have checked this in Azure Portal, the limit is 1000000 now, way beyond 10000. We will increase the limit accordingly.

**Comment by stephybun** [comment]
Closed by #22331
